web/src/tippyjs.js

```diff
@@ -465,3 +465,22 @@ export function initialize() {
         appendTo: () => document.body,
     });
 }
+
+export function show_copied_confirmation($copy_button) {
+    // Display a tooltip to notify the user the message or code was copied.
+    const instance = tippy($copy_button, {
+        placement: "top",
```

Comment:
@amanagr Wouldn't this placement be terrible for reuse? This function is hardcoding the placement to the top. Do you think there should be a bit of changes to accommodate a flexible placement?

Reply:
If we want this function to use a different `placement`, then we can do that in the PR of the feature that needs it.

Reply:
Sounds good!